XmlFormatter/src/Windows/VersionInformation.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Diagnostics;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5553158)